strongbox-testing/strongbox-testing-core/pom.xml

```diff
@@ -105,11 +105,6 @@
             <version>${project.version}</version>
         </dependency>
 
-        <dependency>
```

Comment:
@fuss86 we probably need to drop this from `strongbox-parent` as well

Reply:
yes, correct

Reply:
is there a `strongbox-parent` PR for this?

Reply:
yes, now it is :) https://github.com/strongbox/strongbox-parent/pull/31